transport.go

```diff
@@ -0,0 +1,33 @@
+package gidari
+
+import (
+	"context"
+	"os"
+
+	"github.com/alpine-hodler/gidari/internal/transport"
+)
+
+// TransportFile will construct the transport operation using a configuration YAML file.
+func TransportFile(ctx context.Context, file *os.File) error {
+	var bytes []byte
+
+	_, err := file.Read(bytes)
+	if err != nil {
+		return err
+	}
+
+	cfg, err := transport.NewConfig(bytes)
+	if err != nil {
+		return err
+	}
+
+	return Transport(ctx, cfg)
+}
+
+// Transport will contruct the transport operation using a "transport.Config" object.
+func Transport(ctx context.Context, cfg *transport.Config) error {
```

Comment:
Ah so there is a small problem with this, `transport.Config` is an internal struct. What do you think about adding a struct to this file called "Config" and embed the internal struct into it as a work around for now? 

```go
// Config is the configuration object used to make programatic Transport requests.
type Config struct { transport.Config }

func Transport(ctx context.Context, cfg *Config) error { 
// ...
}
```